### PR TITLE
fix torchbench result analysis bug

### PR DESCRIPTION
**Pull request by Guo-Peilin**

### CODE REVIEW

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh [hunk 1]
```diff
@@ -85,7 +85,11 @@ do
 done
 
 # performance anaysis
-python3 results_anaysis.py -t ${results} -i ${oss_dir} -p ${RELATED_DIFF_PERCENT}
+if [ $1 == "cuda" ]; then
+  python3 results_anaysis.py -t ${results} -i ${oss_dir} -p ${RELATED_DIFF_PERCENT} -f "cuda"
+else
+  python3 results_anaysis.py -t ${results} -i ${oss_dir} -p ${RELATED_DIFF_PERCENT} -f "cpu"
```
Comment (qiuxiafei) [inline pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:91]: nit: just use `-f $1` ?

File: pytorch_blade/benchmark/TorchBench/results_anaysis.py [hunk 2]
```diff
@@ -13,12 +13,17 @@
...
 import pandas as pd
 
-COMPARE_FIELDS = [
+CUDA_COMPARE_FIELDS = [
     "disc (latency)",
     "blade (latency)",
     "dynamo-blade (latency)",
     "dynamo-disc (latency)",
 ]
+CPU_COMPARE_FIELDS = [
```
Comment (qiuxiafei) [inline pytorch_blade/benchmark/TorchBench/results_anaysis.py:22]: nit: how about just pass fields as argument of `analyze_target` ?

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh (hunk 1 above)
Comment (Guo-Peilin) [reply pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:91]: $1 could be "cuda", "cpu" or "aarch64", so I guess we cannot just use `-f $1`

**Review by zzpmiracle — APPROVED** [review]
LGTM, thx！

**Pull request merged**